package/tests/test_api/test_apps.py

```diff
@@ -188,6 +204,111 @@ def assert_example_data(response_data):
     }
 
 
+def assert_example_transcoded_data(response_data):

```

Comment:
I could just put the transcoded edges, and transcoded nodes and check if the json contains it it by writing a function assert_nodes_contain instead of assert_nodes_equal. Does that make sense?

Reply:
Yes it does make sense but doesn't have to be in this PR. Happy to merge as-is.